api/app/Policies/TeamPolicy.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace App\Policies;
+
+use App\Models\Team;
+use App\Models\User;
+use Illuminate\Auth\Access\HandlesAuthorization;
+
+class TeamPolicy
+{
+    use HandlesAuthorization;
+
+    /**
+     * Determine whether the user can view any models.
+     *
+     * @param  \App\Models\User|null  $user
+     * @return \Illuminate\Auth\Access\Response|bool
+     */
+    public function viewAny(User $user = null)
+    {
+        return $user->isAdmin();
```

Comment:
We should probably check if a user exists here before checking if they are admin. Otherwise we get a `Call to a member function isAdmin() on null`

Reply:
I believe that is the role `$user = null` serves, to avoid erroring for guests

it is the same snippet we use elsewhere like

https://github.com/GCTC-NTGC/gc-digital-talent/blob/073d8e1b970fede64705c7eac24b6650d6d3b645/api/app/Policies/DepartmentPolicy.php#L19-L22

Reply:
The issue is that policy methods need to return a `boolean` so they can send a proper error message to the client. 
Right now we get 500 internal server error if they are anonymous instead of a proper 401.

Reply:
ah yes, yes, now I think I got it
nested it in https://github.com/GCTC-NTGC/gc-digital-talent/pull/5548/commits/01d0cb1b447cb15f3a705042fc176a44dab6e20e